app/views/integrations/index.html.erb

```diff
@@ -1,7 +1,53 @@
+<% content_for :sidebar do %>
+  <%= render partial: 'projects/nav', locals: { project: @project } %>
+<% end %>
+
 <% content_for :title_bar do %>
-  <%= render partial: 'projects/nav', locals: {project: @project} %>
+  <div class="project_nav">
+    <%= link_to t('back'), project_path(@project), class: "btn btn-primary btn-sm" %>
+  </div>
 <% end %>
 
 <div class="row">
-  <%= render 'form' %>
+  <div class="col-sm-12 col-md-12">
+    <h3><%= t('projects.integrations') %></h3>
+    <hr />
```

Comment:
I know that this can be a personal taste, but you can use just `<hr>` since HTML 4.0